packages/theme-chalk/src/menu.scss

```diff
@@ -167,7 +167,10 @@
 
     > .#{$namespace}-menu-item,
     > .#{$namespace}-sub-menu > .#{$namespace}-sub-menu__title,
-    > .#{$namespace}-menu-item-group > ul > .#{$namespace}-sub-menu > .#{$namespace}-sub-menu__title {
+    > .#{$namespace}-menu-item-group
+      > ul
+      > .#{$namespace}-sub-menu
+      > .#{$namespace}-sub-menu__title {
```

Comment:
Doesn't seem to be modified here?

Reply:
This is generated after formatting pnpm format

Reply:
I resubmit

Reply:
has been resubmitted @btea

Reply:
I feel that there are still several files after pnpm format. There will be formatted files. I suggest you reformat them once.

Reply:
Ok, I see.